src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -2,23 +2,55 @@ import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { provideMockStore } from '@ngrx/store/testing';
-
+import { of } from 'rxjs';
+import { UserService } from 'src/app/services/user/user.service';
```

Comment:
src should be taken out here

Reply:
@connorpgpmcelroy  this is coming from master merge. But, still fixed it.